base-theme/assets/index.ts

```diff
@@ -11,13 +11,15 @@ import "video.js/dist/video-js.css"
 import { initExternalLinkModal } from "./js/external_link_modal"
 import { initQuotesModalHandler } from "./quote_card_modal_handler"
 import { initVideoDownloadPopup } from "./js/video_download_popup"
+import posthog from "posthog-js"
```

Comment:
I don't think we need any changes to this file now that www and course are initializing Posthog on their own. I tested removing these lines and everything still worked.